Makefile

```diff
@@ -29,6 +31,12 @@ resetdb:
 
 dev-environment: deps initdevdb install-development resetdb
 
+install-cypress:
```

Comment:
What's the motivation behind this new target? I think cypress is already installed as a dep in package.json

Reply:
Hmmm the ubuntu-20 image I'm using doesn't come with some of the cypress dependencies out of the box, so even though yarn is installing cypress it's not enough. I saw in the README that y'all recommend running this in Ubuntu-20, so I figured it'd be good to add a makefile step for installing cypress deps.

Happy to keep this in my fork though, I probably should just be using an ubuntu image that comes with the cypress dependencies anyways.

Reply:
Ah gotcha. I can't remember if the Ubuntu image I'm using came with the deps or not. I'll keep an eye out for this next time I install Ubuntu and if there are missing deps, will add them to the Makefile. Thanks for pointing this out.